components/integrations/confirm_integration/confirm_integration.jsx

```diff
@@ -1,370 +0,0 @@
-import {FormattedMessage} from 'react-intl';
-import {Link} from 'react-router-dom';
-
-import {browserHistory} from 'utils/browser_history';
-import {Constants, ErrorPageTypes} from 'utils/constants';
-import CopyText from 'components/copy_text';
-import BackstageHeader from 'components/backstage/components/backstage_header.jsx';
-import {getSiteURL} from 'utils/url';
-import FormattedMarkdownMessage from 'components/formatted_markdown_message';
-
-export default class ConfirmIntegration extends React.PureComponent {
-    static get propTypes() {
-        return {
-            team: PropTypes.object,
-            location: PropTypes.object,
-            commands: PropTypes.object,
-            oauthApps: PropTypes.object,
-            incomingHooks: PropTypes.object,
-            outgoingHooks: PropTypes.object,
-            bots: PropTypes.object,
-        };
-    }
-
-    constructor(props) {
-        super(props);
-        this.state = {
-            type: (new URLSearchParams(this.props.location.search)).get('type'),
-            id: (new URLSearchParams(this.props.location.search)).get('id'),
-        };
-    }
-
-    componentDidMount() {
-        window.addEventListener('keypress', this.handleKeyPress);
-    }
-
-    componentWillUnmount() {
-        window.removeEventListener('keypress', this.handleKeyPress);
-    }
-
-    handleKeyPress = (e) => {
-        if (e.key === 'Enter') {
-            browserHistory.push('/' + this.props.team.name + '/integrations/' + this.state.type);
-        }
-    }
-
-    render() {
-        let headerText = null;
-        let helpText = null;
-        let tokenText = null;
-
-        const command = this.props.commands[this.state.id];
-        const incomingHook = this.props.incomingHooks[this.state.id];
-        const outgoingHook = this.props.outgoingHooks[this.state.id];
-        const oauthApp = this.props.oauthApps[this.state.id];
-        const bot = this.props.bots[this.state.id];
-
-        if (this.state.type === Constants.Integrations.COMMAND && command) {
-            const commandToken = command.token;
-
-            headerText = (
-                <FormattedMessage
-                    id={'installed_commands.header'}
-                    defaultMessage='Slash Commands'
-                />
-            );
-            helpText = (
-                <p>
-                    <FormattedMessage
-                        id='add_command.doneHelp'
-                        defaultMessage='Your slash command is set up. The following token will be sent in the outgoing payload. Please use it to verify the request came from your Mattermost team (details at <link>Slash Commands</link>).'
-                        values={{
-                            link: (msg) => (
-                                <a
-                                    href='https://developers.mattermost.com/integrate/admin-guide/admin-slash-commands/'
-                                    referrer='noreferrer'
```

Comment:
Typescript is complaining about `referrer`. They are removed based on previous recommendation [here](https://github.com/mattermost/mattermost-webapp/pull/10651#discussion_r907404227)